app/models/chargeback.rb

```diff
@@ -294,13 +294,10 @@ def self.set_chargeback_report_options(rpt, group_by, header, tz)
       rpt.col_order = static_cols + ["display_range"]
       rpt.sortby = static_cols + ["start_date"]
     end
-    rpt.col_order.each do |c|
-      if c == tag_col
-        rpt.headers.push(Dictionary.gettext(header, :type => :column, :notfound => :titleize)) if header
-      else
-        rpt.headers.push(Dictionary.gettext(c, :type => :column, :notfound => :titleize))
-      end
 
+    rpt.col_order.each do |c|
+      header_column = (c == report_tag_field && header_for_tag) ? header_for_tag : c
+      rpt.headers.push(Dictionary.gettext(header_column, :type => :column, :notfound => :titleize))
```

Comment:
This will push when `header_for_tag == nil`

That's a behavior change. I am not sure, what is the impact of this. Can you please take a look and confirm, that is Ok?